packages/trader/src/sass/app/_common/layout/trader-layouts.scss

```diff
@@ -393,11 +400,15 @@ $FLOATING_HEADER_HEIGHT: 41px;
             }
         }
     }
-    @include mobile {
+
+    @include mobile-screen {
         flex-direction: column;
-        min-height: calc(100vh - 48px);
         padding: 0;
     }
+
+    @include mobile-or-tablet-screen {
+        min-height: calc(100vh - 48px);
```

Comment:
just a question: where do we take these 48px ? :)

Reply:
The value was already there, i just move it from mobile only to mobile-or-tablet-screen :))

Reply:
But it is substracting the view height with the header